corehq/apps/hqadmin/admin.py

```diff
@@ -13,6 +14,11 @@ class ESRestorePillowCheckpointsAdmin(admin.ModelAdmin):
         'seq',
         'seq_int',
     ]
+    search_fields = [
+        'checkpoint_id',
+    ]
+
+    ordering = ['-date_updated']
```

Comment:
fwiw that's already defined on the model https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqadmin/models.py#L92